common/src/main/resources/db/changelog/v001/schema.sql

```diff
@@ -40,14 +40,14 @@ ALTER TABLE sponsor ADD CONSTRAINT "fk_sponsor_to_sponsor_parent" FOREIGN KEY (p
 --changeset spathiyil:AB2D-291-CreateTable-contract failOnError:true 
 CREATE TABLE contract
 (
-    id                  BIGINT              NOT NULL,
+    id                  BIGSERIAL              NOT NULL,
```

Comment:
Unsure of the purpose of this change. We should keep the primary keys consistent across, i.e. use `BIGINT` coupled with a sequence for all.

Reply:
So in https://github.com/CMSgov/ab2d/pull/110/files#diff-d72ce5f81287c43e6ea1c18ac61182c2 after removing the contract ID sequence and using serial for the ID in contract table, I’m not seeing the issue that @Satheesh Pathiyil saw earlier with the missing_sequence mentioned here https://github.com/CMSgov/ab2d/pull/31#discussion_r341251018 and my e2e tests run through the app quite a bit. I’m guessing something changed in the application/db so that it’s no longer an issue. 

I was having issues with setting up contracts manually with SQL inserts, the stress testing is some of the new stuff I'm currently working on while we're looking for new tasks.

Reply:
Stress testing is a whole different beast; it will likely need to go into a separate module.